exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitExtensionTest.java

```diff
@@ -27,18 +36,14 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.jupiter.api.io.TempDir;
 import org.junit.platform.engine.TestExecutionResult;
 import org.junit.platform.testkit.engine.EngineExecutionResults;
 import org.junit.platform.testkit.engine.EngineTestKit;
 import org.junit.platform.testkit.engine.Event;
 import org.junit.platform.testkit.engine.Events;
 
-class TestKitExtensionTest extends TestKitTestWithArtifactsCreated {
-
-  private static final TestKit.Builder defaultBuilder = TestKit.builder()
```

Comment:
As all the test classes below are invoked explicitly from within this test class, could we create the directory and an artifact here and once, so that that code isn't duplicated?